src/main/java/com/artipie/rpm/benchmarks/RpmBench.java

```diff
@@ -71,13 +79,52 @@ public void setup() {
         if (RpmBench.BENCH_DIR == null) {
             throw new IllegalStateException("BENCH_DIR environment variable must be set");
         }
-        this.storage = new FileStorage(Paths.get(RpmBench.BENCH_DIR));
+        this.storage = new InMemoryStorage();
+        final Storage src = new FileStorage(Paths.get(RpmBench.BENCH_DIR));
+        RpmBench.sync(src, this.storage);
+    }
+
+    @Setup(Level.Iteration)
+    public void setupIter() {
+        this.storage.delete(new Key.From("/repodata"));
```

Comment:
@g4s8 keys with empty parts are not allowed, see `com/artipie/asto/Key.java:172`, please, list items by `repodata` key and remove every item

Reply:
@olenagerasimova thanks, fixed